### DISCUSSION TITLE
6 Days after approval but still didn't get student benefit

### DISCUSSION THREAD

**Discussion by DrFrankSChen** [community/community#128457] [GitHub Education] [answered]
### Hi everyone,

I have got approved since Jun 7, but until now, I haven't got the benefit. When heading to Github Copilot, it still asks me to buy it. I checked the FAQ and It is said that the benefit should be available in 5 days. 
<img width="329" alt="Screenshot 2024-06-12 at 3 37 20 PM" src="https://github.com/community/community/assets/37143311/823a40ac-5360-4f09-bef1-b5c65dfad220">
<img width="974" alt="Screenshot 2024-06-12 at 3 37 52 PM" src="https://github.com/community/community/assets/37143311/5b259219-d461-412a-8107-32f0e6072612">

**Comment by bfuerholz** [comment]
Hi MrFrankChen,

I’m sorry to hear that you haven’t received your student benefits yet. Here are a few steps you can take to resolve the issue:

Check Approval Email: Ensure you have received a confirmation email stating your approval for the GitHub Student Developer Pack. This email should detail the benefits you’re entitled to.

Verify Eligibility: Double-check that your GitHub account meets all the eligibility requirements for the Student Developer Pack. Sometimes, small discrepancies in your provided information can cause delays.

Wait Time: Although the FAQ mentions a 5-day waiting period, there might be occasional delays. Given that today is June 13, it has been 6 days, so you are right to follow up (my approval took almost 2 weeks).

Contact Support: If you still don’t see the benefits, you should contact GitHub Education support directly. Here’s how:

- Visit the [GitHub Education Support page](https://support.github.com/contact?tags=education).
- Provide your approval ID (#128457) and a brief description of your issue.
- Include any relevant screenshots, such as the approval email and the GitHub Copilot page asking you to purchase.


I hope this helps and that you get your benefits soon. If you have any further questions, feel free to ask!

Best regards

**Comment by davevad93** [comment, marked as answer]
Hi @MrFrankChen ,

That's odd, considering your benefits are already available, so try the following:

1. Open `VS Code`, download the `GitHub Copilot` extension if you haven't did it yet and log in with your GitHub account. If you can use `Copilot` it means everything is ok.
2. If the above solution doesn't work you should contact with support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

**Reply by DrFrankSChen** [reply]
Hi @davevad93 ,
Thank you for your reply. I tried this two days ago, but it didn't work. However, I just tried it again, and it works now. I guess the benefit was not available at the time I tired before.
